### PR TITLE
Added questions to html-quiz

### PR DESCRIPTION
**Pull request by vrashankshetty**
## Summary of changes

Added questions to html quiz

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/html-quiz.ts [hunk 1]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
     Explanation:
       "Hypertext in HTML refers to text (often with embeds such as images, too) that is organized in order to connect web pages to one another.",
     Link: "https://www.freecodecamp.org/news/what-is-html-definition-and-meaning/"
+  }, {
+    Question: "Which is not a block-level element in HTML?",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1663]:
```suggestion
    Question: "In HTML, which of the following elements is NOT a block-level element?",
```

File: src/data/html-quiz.ts [hunk 2]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Answer: "<span>",
+    Distractor1: "<h2>",
+    Distractor2: "<p>",
+    Distractor3: "<span>",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1667]: distractors are for wrong answers only

```suggestion
    Distractor3: "<div>",
```

File: src/data/html-quiz.ts [hunk 3]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Explanation:
+      "<span> is an inline element. It is used to apply styles to a specific part of the text within a block-level element without breaking the flow of the content",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1669]:
```suggestion
      "The span element is a generic inline container that can be used for grouping content and styling purposes.",
```

File: src/data/html-quiz.ts [hunk 4]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Link: "https://www.freecodecamp.org/news/inline-elements-and-block-elements-in-html-explained/"
+  }, {
+    Question: "Which HTML5 feature allows for the storage of data on the client-side, even after the browser is closed?",
+    Answer: "Local Storage",
+    Distractor1: "WebSockets",
+    Distractor2: "Local Storage",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1675]: distractors are for wrong answers only

```suggestion
    Distractor2: "navigator",
```

File: src/data/html-quiz.ts [hunk 5]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Distractor3: "Session Storage",
+    Explanation:
+      " Local Storage is a client-side storage mechanism introduced in HTML5. It allows web applications to store data in a key-value pair format on the user's browser.",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1678]:
```suggestion
      "Local Storage is a client-side storage mechanism introduced in HTML5. It allows web applications to store data in a key-value pair format on the user's browser.",
```

File: src/data/html-quiz.ts [hunk 6]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Link: "https://www.freecodecamp.org/news/how-web-storage-works/"
+  },{
+    Question: "Which is the default type of input field if type is not specified",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1681]:
```suggestion
    Question: "In HTML, which of the following is the default type for the input field?",
```

File: src/data/html-quiz.ts [hunk 7]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Answer: "text",
+    Distractor1: "radio",
+    Distractor2: "text",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1684]: distractors are for only wrong answers

```suggestion
    Distractor2: "color",
```

File: src/data/html-quiz.ts [hunk 8]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Distractor3: "email",
+    Explanation:
+      "If the type attribute is not specified in an HTML <input> element, the default type is text",
+    Link: "https://www.freecodecamp.org/news/text-box-in-html-the-input-field-html-tag/"
+  },{
+    Question: "In HTML,how can you create a range input that allows users to select values from 1 to 100 with step increments of 5?",
+    Answer: '<input type="range" min="1" max="100" step="5">',
+    Distractor1: '<input type="range" min="1" max="100" step="5">',
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1692]: distractors are for wrong answers only

```suggestion
    Distractor1: '<input type="range" min="34" max="100" step="15">',
```

File: src/data/html-quiz.ts [hunk 9]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Distractor2: '<input type="range" min="1" max="100" increment="5">',
+    Distractor3: '<input type="slider" min="1" max="100" step="5">',
+    Explanation:
+      '<input> element with the type attribute set to "range." To define the range and the step increments, you use the min and max attributes to specify the minimum and maximum values,and the step attribute to set the step increment.',
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1696]:
```suggestion
      'To define the range and the step increments for an input element, use the min and max attributes to specify the minimum and maximum values, and the step attribute to set the step increment.',
```

File: src/data/html-quiz.ts [hunk 10]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Question: "Which is the default type of input field if type is not specified",
+    Answer: "text",
+    Distractor1: "radio",
+    Distractor2: "text",
+    Distractor3: "email",
+    Explanation:
+      "If the type attribute is not specified in an HTML <input> element, the default type is text",
+    Link: "https://www.freecodecamp.org/news/text-box-in-html-the-input-field-html-tag/"
+  },{
+    Question: "In HTML,how can you create a range input that allows users to select values from 1 to 100 with step increments of 5?",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1690]:
```suggestion
    Question: "In HTML, how can you create a range input that allows users to select values from 1 to 100 with step increments of 5?",
```

File: src/data/html-quiz.ts [hunk 11]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Link: "https://www.w3.org/wiki/Html/Elements/input/range"
+  },{
+    Question: "What HTML5 attribute is used to associate a <label> element with a specific form control?",
+    Answer: 'for',
+    Distractor1: 'control',
+    Distractor2: 'link',
+    Distractor3: 'for',
+    Explanation:
+      'In HTML, the for attribute is used in conjunction with the <label> element to associate the label with a specific form control (such as an <input> element). This association is important for improving the usability and accessibility of web forms.',
+    Link: "https://www.freecodecamp.org/news/html-label-label-tag-example/#:~:text=You%20use%20the%20HTML%20%3Clabel,elements%20inside%20a%20form%20element."
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1706]:
```suggestion
    Link: "https://www.freecodecamp.org/news/html-label-label-tag-example"
```

File: src/data/html-quiz.ts [hunk 12]
```diff
@@ -1659,6 +1659,53 @@ const htmlQuiz = [
...
+    Distractor3: '<input type="slider" min="1" max="100" step="5">',
+    Explanation:
+      '<input> element with the type attribute set to "range." To define the range and the step increments, you use the min and max attributes to specify the minimum and maximum values,and the step attribute to set the step increment.',
+    Link: "https://www.w3.org/wiki/Html/Elements/input/range"
+  },{
+    Question: "What HTML5 attribute is used to associate a <label> element with a specific form control?",
+    Answer: 'for',
+    Distractor1: 'control',
+    Distractor2: 'link',
+    Distractor3: 'for',
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1703]: distractors are for wrong answers only

```suggestion
    Distractor3: 'date',
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by vrashankshetty** [comment]
> Thank you for your contribution!
> 
> Please review the requested changes. Once those changes have been implemented, then we can merge your PR into main. 👍

Committed all changes.
Thank you Ma'am

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**